Tools/TestWebKitAPI/WebTransportServer.mm

```diff
@@ -72,36 +71,25 @@
     ASSERT(parameters);
     nw_parameters_set_server_mode(parameters.get(), true);
 
-    RetainPtr webtransportOptions = adoptNS(nw_webtransport_create_options());
-    ASSERT(webtransportOptions);
-    nw_webtransport_options_set_is_unidirectional(webtransportOptions.get(), false);
-    nw_webtransport_options_set_is_datagram(webtransportOptions.get(), true);
-
     RetainPtr listener = adoptNS(nw_listener_create(parameters.get()));
 
-    nw_listener_set_new_connection_group_handler(listener.get(), [data = m_data, webtransportOptions = webtransportOptions] (nw_connection_group_t connectionGroup) {
-        ASSERT(!data->connectionGroup);
-        data->connectionGroup = connectionGroup;
-
-        nw_connection_group_set_state_changed_handler(connectionGroup, [data = data, webtransportOptions = webtransportOptions](nw_connection_group_state_t state, nw_error_t error) {
-            if (state == nw_connection_group_state_ready) {
-                // We need to peel off the datagram connection.
-                nw_connection_t datagramConnection = nw_connection_group_extract_connection(data->connectionGroup.get(), nil, webtransportOptions.get());
-                data->connections.append(datagramConnection);
-                data->coroutineHandles.append(data->connectionHandler(datagramConnection).handle);
-                nw_connection_set_queue(datagramConnection, dispatch_get_main_queue());
-                nw_connection_start(datagramConnection);
-            }
+    nw_listener_set_new_connection_group_handler(listener.get(), [
+        data = m_data
+    ] (nw_connection_group_t incomingConnectionGroup)
+    {
```

Comment:
This should be on the previous line.  In fact, could you put the last 4 lines on 1 line?

nw_listener_set_new_connection_group_handler(listener.get(), [data = m_data] (nw_connection_group_t incomingConnectionGroup) {